Resources/Textures/Objects/Misc/flesh.rsi/meta.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "https://github.com/tgstation/tgstation/commit/15bf91049e33979a855995579b48592e34bcdd8c",
```

Comment:
Make it like `Taken from tgstation at <link>`

Reply:
The original code was like this

Reply:
Wdym the original? Did you take it from somewhere?

Reply:
this is a copy of the floor from the flesh anomaly